free-programming-books-tr.md

```diff
@@ -96,6 +94,7 @@
 * [GNU Bash Başvuru Kılavuzu](http://belgeler.org/bashref/bashref.html)
 * [Linux Belgeleri](http://belgeler.org/howto/howtos.html)
 * [Linux Sistem Yöneticisinin Kılavuzu](http://belgeler.org/sag/sag.html)
+* [Gnu Linux Komutlari](https://www.fullportal.org/GNULINUX/Komutlar/GNULINUXKOMUTLAR.pdf)
```

Comment:
```suggestion
* [GNU Bash Başvuru Kılavuzu](http://belgeler.org/bashref/bashref.html)
* [Gnu Linux Komutlari](https://www.fullportal.org/GNULINUX/Komutlar/GNULINUXKOMUTLAR.pdf) (PDF)
* [Linux Belgeleri](http://belgeler.org/howto/howtos.html)
* [Linux Sistem Yöneticisinin Kılavuzu](http://belgeler.org/sag/sag.html)
```

2 part fix here:

1.  Alphabetical order
2. Indicating it is a PDF extension with (PDF)